### ISSUE TITLE
2 new pages

### ISSUE THREAD

**Issue by parthivppatel** [Puzer-03/portfolio#13]
Create 2 more section: 

1. Education: degree, school, about, startdate, end date
2. Experience: type, organization, about, startdate, end date

there will be a check box of currently ongoing which will remove end date column and set to current in the database

**Comment by Puzer-03** [comment]
In the table of both forms please reference user id

**Comment by Puzer-03** [comment]
Have you started working on this because we have to show these both on the portfolio page

**Comment by parthivppatel** [comment]
bro I already started working on it and one page is page is ready but I got some issues in it so I not push the code yesterday night

**Comment by parthivppatel** [comment]
note that I done education module but in checkbox I am make the multiple logic for it but it is not applied on it , the problem is 
in checkbox  if we use checked=true checkbox always true but  it not gives us false condition (because bydefault checkbox is not tick(d= means false)
now the database I store 0 and 1 for it and I fetch data for it but like all field data mapping in checkbox is not possible

@hetparekh21 or @Puzer-03  if anyone found solution please fix

**Comment by parthivppatel** [comment]
and in start_date and end_date plase remove placeholders and do field clean when first time , i find it but not got solution of it

**Comment by Puzer-03** [comment]
can you please elaborate both of your queries

**Comment by parthivppatel** [comment]
please see the frontend and test if not understand call me

**Comment by hetparekh21** [comment]
Guys I don't think this is the way we should build the education page , I think it should accept multiple education sets (degree,date,institution etc) Like SKILLS or CATEGORIES .
@parthivppatel @Puzer-03

**Comment by Puzer-03** [comment]
yeah, that's what we discussed we will give a add button and show the following details on a table similar to skill and category section

**Comment by parthivppatel** [comment]
we are discuss on tomorrow how the frontend is design in education and experience page

**Comment by Puzer-03** [comment]
There are some changes so we'll discuss it today

**Comment by Puzer-03** [comment]
any updates?
